docs/agent-skills/apollo-client/references/state-management.md

```
@@ -0,0 +1,412 @@
+# State Management Reference
+
+## Table of Contents
+
+- [Reactive Variables](#reactive-variables)
+- [Local-Only Fields](#local-only-fields)
+- [Type Policies for Local State](#type-policies-for-local-state)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)